src/seedu/addressbook/commands/FindByTagCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.*;
```

Comment:
Should import specifically used instead of wildcarding

Reply:
Thanks @eugenefdw for spotting this. I have changed it accordingly.